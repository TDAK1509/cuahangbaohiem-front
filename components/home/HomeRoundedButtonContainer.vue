<template>
  <div class="home-rounded-button-container">
    <slot />
  </div>
</template>

<script>
export default {
  name: "HomeRoundedButtonContainer"
};
</script>

<style lang="scss" scoped>
@import "~assets/scss/_breakpoints";

.home-rounded-button-container {
  display: grid;
  grid-template-columns: 1fr 1fr 1fr 1fr;
  justify-items: center;
  row-gap: 3rem;
  max-width: 900px;
  margin: auto;
}

@media only screen and (max-width: $mobile) {
  .home-rounded-button-container {
    grid-template-columns: 1fr 1fr;
    row-gap: 1.5rem;
  }
}
</style>
