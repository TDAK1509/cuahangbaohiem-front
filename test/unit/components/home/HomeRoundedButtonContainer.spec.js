import { mount } from "@vue/test-utils";
import HomeRoundedButtonContainer from "@/components/home/HomeRoundedButtonContainer.vue";

describe("HomeRoundedButtonContainer.vue", () => {
  it("matches snapshot", () => {
    const wrapper = mount(HomeRoundedButtonContainer, {
      slots: {
        default: "<div>Slot content</div>"
      }
    });
    expect(wrapper.element).toMatchSnapshot();
  });
});
