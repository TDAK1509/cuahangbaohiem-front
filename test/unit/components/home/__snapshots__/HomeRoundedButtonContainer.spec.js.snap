// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HomeRoundedButtonContainer.vue matches snapshot 1`] = `
<div
  class="home-rounded-button-container"
>
  <div>
    Slot content
  </div>
</div>
`;
